h/templates/client/thread.html

```diff
@@ -45,3 +45,7 @@
       ng-show="vm.shouldShowAsReply()">
   </li>
 </ul>
+
+<footer class="thread-anchor-notice" ng-if="vm.container.message.$orphan">
```

Comment:
Here's the problem. This got messed up in some rebasing or refactoring. I may have pushed it with this problem to begin with.

The flag is `$anchored` not `$orphan`, and the meaning is inverted.